docs: Added Docs for FAQs (#111)
DenverCoder1/readme-typing-svg: docs/faq.md

> # Versions of the Readme Typing SVG

Reviewer: If this is posed as a question, it could even be included below the FAQs header.
```suggestion
# How do I include Readme Typing SVG in my Readme?

Markdown files on GitHub support embedded images using Markdown or HTML.
```

> Image version:

Reviewer: Both are image, this one is Markdown syntax
```suggestion
Markdown:
```

```html
[![Typing SVG](https://readme-typing-svg.herokuapp.com/?lines=First+line+of+text;Second+line+of+text)](https://git.io/typing-svg)
```

> HTML version:

Reviewer: ```suggestion
HTML:
```

```html

>   <img src="https://readme-typing-svg.herokuapp.com/?lines=Type+messages+everywhere!;Add+a+bio+to+your+profile!;Add+a+description+to+your+repo!;Make+your+readme+stand+out!&font=Fira%20Code&center=true&width=380&height=50">

Reviewer: While the link is not required for it to function, it would make it equivalent to the markdown example

```suggestion
<a href="https://git.io/typing-svg"><img src="https://readme-typing-svg.herokuapp.com/?lines=First+line+of+text;Second+line+of+text"></a>
```

```

# FAQs

> ## 1. Help — how do I center the Image/Typing SVG?

Reviewer: This can be made more concise.
```suggestion
## How do I center the image on the page?
```

We can do so by *using the HTML version*, and then wrapping it by the HTML attribute of `align="center"`.

You can do so by:

```html
<p align="center">
    <img src="https://readme-typing-svg.herokuapp.com/?lines=Type+messages+everywhere!;Add+a+bio+to+your+profile!;Add+a+description+to+your+repo!;Make+your+readme+stand+out!&font=Fira%20Code&center=true&width=380&height=50">
</p>
```

> >Note: Why do we have to use the HTML version? It is because we can only use the `align="center"` on a HTML attribute.

Reviewer: I think this note is not needed since it is explained above to use the HTML version.

```suggestion
```
